ci-operator/config/openshift/jenkins/openshift-jenkins-master.yaml

```diff
@@ -39,7 +39,7 @@ images:
     src:
       paths:
       - destination_dir: .
-        source_path: /go/src/github.com/openshift/jenkins/agent-nodejs-10/.
+        source_path: /go/src/github.com/openshift/jenkins/agent-nodejs-8/.
```

Comment:
This should be agent-nodejs-12 instead, with a dockerfile_path of Dockerfile.rhel8.